pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+[//]: # (You should also always follow the Pull Request Template when opening a PR. Following the pull request template is an important step to ensure that the information is presented in a clean and readable way.)
+[//]: # (Before a Pull Request is merged it will go through a couple of checks to makes sure the fix works and is compatible.)
+[//]: # (The PR will be built by the CI's.)
+[//]: # (You must have your PR tested to ensure that it properly works. We recommend that you at least test your own PR, and you to ask others for help to test your changes.)
```

Comment:
(You must test your own PR to the best of your capabilities, and we encourage you to seek out others to help you in the effort to best prevent bugs and other anomalies from making their way into the core.)